superbench/benchmarks/micro_benchmarks/nvbandwidth.py

```diff
@@ -83,6 +83,30 @@ def add_parser_arguments(self):
             help='Iterations of the benchmark. Default is 3.',
         )
 
+        self._parser.add_argument(
+            '--numa',
+            type=int,
+            required=False,
+            help='The index of numa node.',
+        )
+
+    def __get_arguments_from_env(self):
```

Comment:
This is a very good function, can we put a base empty function inside the base class?